frontend/src/api/network/notebooks.ts

```diff
@@ -10,74 +10,11 @@ import {
 import * as _ from 'lodash';
 import { NotebookModel } from '../models';
 import { K8sStatus, NotebookKind } from '../../k8sTypes';
-import {
-  NotebookAffinity,
-  NotebookResources,
-  NotebookToleration,
-  NotebookTolerationSettings,
-} from '../../types';
 import { usernameTranslate } from '../../utilities/notebookControllerUtils';
 import { EnvironmentFromVariable, StartNotebookData } from '../../pages/projects/types';
 import { ROOT_MOUNT_PATH } from '../../pages/projects/pvc/const';
 import { translateDisplayNameForK8s } from '../../pages/projects/utils';
-
-const assembleNotebookAffinityAndTolerations = (
-  notebookResources: NotebookResources,
-  gpus: number,
-  tolerationSettings?: NotebookTolerationSettings,
-): {
-  affinity: NotebookAffinity;
-  tolerations: NotebookToleration[];
-  resources: NotebookResources;
-} => {
-  let affinity: NotebookAffinity = {};
-  const tolerations: NotebookToleration[] = [];
```

Comment:
FWIW -- we'll have a conflict here -- I moved some of the toleration stuff around in #903.